index.d.ts

```diff
@@ -60,3 +60,24 @@ console.log(await getPort({port: portNumbers(3000, 3100)}));
 ```
 */
 export function portNumbers(from: number, to: number): Iterable<number>;
+
+/**
+Clear the internal cache of locked ports when you want your results to be unaffected by previous calls.
+
+@example
+```
+import getPort, {clearLockedPorts} from 'get-port';
+
+console.log(await getPort({prot: [3000, 3001, 3002]}));
+//=> 3000
+
+console.log(await getPort({prot: [3000, 3001, 3002]}));
+//=> 3001
+
+// If you want your results to be unaffected by previous calls, clear the cache.
+clearLockedPorts();
+console.log(await getPort({prot: [3000, 3001, 3002]}));
+//=> 3000
+```
+ */
```

Comment:
Please don't resolve when it's not fixed.
```suggestion
*/
```